common/defs/bitcoin/syscoin.json

```diff
@@ -9,9 +9,9 @@
   "decimals": 8,
   "address_type": 63,
   "address_type_p2sh": 5,
-  "maxfee_kb": 10000000,
-  "minfee_kb": 1000,
-  "signed_message_header": "Syscoin Signed Message:\n",
+  "maxfee_kb": 20000000,
+  "minfee_kb": 10000,
+  "signed_message_header": "Bitcoin Signed Message:\n",
```

Comment:
This is unacceptable - this will allow an attacker to create a Bitcoin signed message while the user thinks they are signing the Syscoin message.

Unless this is fixed we can't merge this.

Reply:
What is the attack scenario? There are other coins that also do this which are not Bitcoin btw.

Reply:
I suggest you change the value to `"Syscoin Signed Message:\n"` like it was before https://github.com/syscoin/syscoin/commit/849c9362451d8aa70bd55b464cba810e2d9fb03d

Having it set to Bitcoin ... is non-sense.

Reply:
Done